packages/main/src/plugin/index.spec.ts

```
@@ -423,13 +423,48 @@ test('Pull image creates a task', async () => {
   expect(registeredCallback).not.equal(defaultCallback);
   registeredCallback({ id: 'pullEvent1' } as PullEvent);
   expect(createTaskSpy).toHaveBeenCalledOnce();
-  expect(createTaskSpy).toHaveBeenCalledWith({ title: `Pulling registry.com/repo/image:latest`, action: undefined });
+  expect(createTaskSpy).toHaveBeenCalledWith({
+    title: `Pulling registry.com/repo/image:latest`,
```

Biome (2.4.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)